packages/ruleset/src/functions/property-description.js

```diff
@@ -9,6 +9,11 @@ module.exports = function(schema, _opts, { path }) {
 };
 
 function propertyDescription(schema, path) {
+  // If "schema" is a $ref, that means it didn't get resolved
+  // properly (perhaps due to a circular ref), so just ignore it.
+  if (schema.$ref) {
+    return [];
+  }
```

Comment:
This is just a question, but shouldn't this throw at least a warning? Or it would be duplicated if it’s caused by circular reference, and that won’t be good?

Reply:
Yeah, the assumption is that the circular-refs rule will catch it and we don't want duplicate warnings/errors to be displayed.